lib/src/arc_text_painter.dart

```diff
@@ -13,40 +13,52 @@ class ArcTextPainter {
     double initialAngle = 0,
     Direction direction = Direction.clockwise,
     Placement placement = Placement.outside,
+    double stretchAngle,
+    double interLetterAngle,
   })  : assert(radius != null, 'radius should not be null'),
         assert(text != null, 'text should not be null'),
         assert(textStyle != null, 'textStyle should not be null'),
         assert(alignment != null, 'alignment should not be null'),
         assert(initialAngle != null, 'initialAngle should not be null'),
         assert(direction != null, 'direction should not be null'),
         assert(placement != null, 'placement should not be null'),
+        assert(!(stretchAngle != null && interLetterAngle != null),
```

Comment:
I would say the inversed condition would be slightly more readable.

```suggestion
        assert(stretchAngle == null || interLetterAngle == null,
```